spec/components/search/thumbnail_component_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Search::ThumbnailComponent, type: :component do
+  let(:component) { described_class.new(result:) }
+  let(:rendered) { render_inline(component) }
+  let(:result) do
+    double(SearchResults::Item, # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
Why not an `instance_double`?

Reply:
I was following what was done in `item_result_component_spec`, but this highlighted some other cruft I missed as well, fixed.